go/test/endtoend/sharding/base_sharding.go

```diff
@@ -139,8 +139,12 @@ func CheckBinlogPlayerVars(t *testing.T, vttablet cluster.Vttablet, sourceShards
 	tabletVars := vttablet.VttabletProcess.GetVars()
 
 	assert.Contains(t, tabletVars, "VReplicationStreamCount")
+	// DEPRECATED
 	assert.Contains(t, tabletVars, "VReplicationSecondsBehindMasterMax")
 	assert.Contains(t, tabletVars, "VReplicationSecondsBehindMaster")
+
+	assert.Contains(t, tabletVars, "VReplicationSecondsBehindPrimaryMax")
```

Comment:
Suggestion: VReplicationLagSecondsMax.
I have already replaced `SecondsBehindMaster` with `ReplicationLagSeconds` in `RealTimeStats`, so this will keep it consistent with that (and avoids master/primary altogether).